### PR TITLE
Training PyTorch models on Cloud AI Platform

### PR DESCRIPTION
**Pull request by RajeshThallam**
This pull request is to add notebook and scripts related to an upcoming blog series on training and deploying PyTorch models on Google Cloud AI Platform.

### CODE REVIEW

**Comment by ivanmkc** [comment]
@RajeshThallam Is this for CAIP?

If so, can you move it to: https://github.com/GoogleCloudPlatform/ai-platform-samples/tree/master/ai-platform/tutorials/unofficial/<your tutorial name>?

If for uCAIP, move it: https://github.com/GoogleCloudPlatform/ai-platform-samples/tree/master/ai-platform-unified/tutorials/unofficial/<your tutorial name>

**Comment by ivanmkc** [comment]
As for the code, I won't review as I'm just maintaining the repo. Feel free to find someone who knows your tutorial to review from your team.

I'll approve once tests pass and you move everything.

**Comment by RajeshThallam** [comment]
@ivanmkc This is for CAIP. I am fine to move it to the new directory but we wanted to align with existing notebook examples in https://github.com/GoogleCloudPlatform/ai-platform-samples/tree/master/notebooks/samples/pytorch

Will rest of the notebooks there will be moved as well?

File: ai-platform/notebooks/unofficial/tutorials/pytorch/sentiment_classification/pytorch-text-classification-caip-training.ipynb
Comment (amygdala) [inline ai-platform/notebooks/unofficial/tutorials/pytorch/sentiment_classification/pytorch-text-classification-caip-training.ipynb:44]: link typo here (causing rendering glitch)

File: ai-platform/notebooks/unofficial/tutorials/pytorch/sentiment_classification/pytorch-text-classification-caip-training.ipynb
Comment (amygdala) [inline ai-platform/notebooks/unofficial/tutorials/pytorch/sentiment_classification/pytorch-text-classification-caip-training.ipynb:488]: Is this missing some text?

File: ai-platform/notebooks/unofficial/tutorials/pytorch/sentiment_classification/pytorch-text-classification-caip-training.ipynb
Comment (amygdala) [inline ai-platform/notebooks/unofficial/tutorials/pytorch/sentiment_classification/pytorch-text-classification-caip-training.ipynb:89]: You probably need some info about requesting T4 quota, as it's unlikely they'll have this out of the box.

**Comment by RajeshThallam** [comment]
@ivanmkc Can you help with merging the PR? Review comments from @amygdala are taken care of.

**Review by ivanmkc — APPROVED** [review]
LGTM

**Comment by ivanmkc** [comment]
Please add CODEOWNERS file

**Comment by RajeshThallam** [comment]
Added CODEOWNERS file

**Pull request merged**
